src/Features/LanguageServer/Protocol/Extensions/ProtocolConversions.cs

```diff
@@ -346,7 +348,6 @@ public static LSP.SymbolKind GlyphToSymbolKind(Glyph glyph)
                 case Glyph.DelegateProtected:
                 case Glyph.DelegatePrivate:
                 case Glyph.DelegateInternal:
-                    return LSP.SymbolKind.Function;
```

Comment:
Add comment here saying we aren't distinguishing?

Reply:
I don't think a comment here makes sense unless you know the history. Looked at as a whole, this file simply maps Delegate to Method and vice versa.